docs/src/Experimental/divisors.md

```diff
@@ -0,0 +1,71 @@
+```@meta
+CurrentModule = Oscar
+```
+
+```@setup oscar
+using Oscar
+```
+
+```@contents
+Pages = ["divisors.md"]
+```
+
+# Divisors
+
+In order to consider divisors on curves, we restrict our attention to smooth and irreducible curves.
+
+Let ``C`` be an affine or projective plane curve defined by an irreducible equation ``F``. Then any polynomial function ``G`` which is not divisible by ``F`` will vanish on ``C`` only at finitely many points. A way to encode these points together with their intersection multiplicities is to consider a divisor. A divisor on a curve is a formal finite sum of points of the curve with integer coefficients. A natural operation of addition can be defined on the set of divisors of a curve, which turns it into an Abelian group.
```

Comment:
Please wrap this line, I'd suggest to aim for lines which are something between 70-90 characters wide.